src/main.rs

```diff
@@ -114,17 +121,29 @@ impl EventHandler for Handler {
                             .guild(guild_id)
                             .expect("Could not find guild in cache.");
 
-                        let channel_id = match guild
-                            .voice_states
-                            .get(&config.discord_user_id.into())
-                            .and_then(|voice_state| voice_state.channel_id)
-                        {
-                            Some(channel_id) => channel_id,
-                            None => {
-                                println!("Could not find user in VC.");
-                                continue;
+                        // Check if any of the admins are in the VC
+                        let user_list = guild.voice_states;
+                        
+                        let mut channel_ids: Vec<Option<id::ChannelId>> = Vec::new();
```

Comment:
Why do you use here an vector if in fact you only need one single channel_id later on?

Add the `is_some` check of line 142 as a condition to the if in 130 and only assign if a valid channel was found.
The var channel_id should then be an `Option<id::ChannelId>` initialized to `None`.

Then you can check if it is still None after your loop, if that's the case log that the 'admin' was not found. If it is some, join the channel.